frontend/src/app/testQueue/AoEForm/AoEModalForm.jsx

```diff
@@ -69,6 +75,11 @@ const AoEModalForm = ({
     </div>
   );
 
+  function sendSms() {
+    // TODO: implement SMS sending
+    setSmsSuccess(true);
```

Comment:
This is probably an async operation so keep in mind whether that will break the existing logic. Based on line 123 I think it should work but the Send button will stay active so you might need to disable the button here once the async operation arrives.

Reply:
Good catch @dmethvin-gov!

I'll account for the async considerations and loading/error states in a separate issue #831